vizier/datastore/mimir/store.py

```diff
@@ -98,7 +98,10 @@ def create_dataset(self, columns, rows, human_readable_name = None, annotations=
                     name_in_rdb=col.name
                 )
             )
-            colSql = colSql + ', ' + col.name + ' AS ' + col.name
+            if colSql == '':
+                colSql = col.name + ' AS ' + col.name
+            else:
+                colSql = colSql + ', ' + col.name + ' AS ' + col.name
```

Comment:
Again, it would be good to avoid SQL if possible.  If not possible, are the column names at least being escaped (I see they're being sanitized)?